csrankings-b.csv

```diff
@@ -728,6 +728,7 @@ Branislav Bosanský,Czech Technical University,https://sites.google.com/site/bbo
 Branislav Bošanský,Czech Technical University,https://sites.google.com/site/bbosansky,jltCTOwAAAAJ
 Brendan A. Juba,Washington University in St. Louis,http://www.cse.wustl.edu/~bjuba,5wppdUoAAAAJ
 Brendan D. McKay,Australian National University,https://cs.anu.edu.au/people/brendan-mckay,3nV4EKwAAAAJ
+Brendan David-John,Virginia Tech,https://brendandavidjohn.app,YFxRRDUAAAAJ

```

Comment:
Need to add alias `Brendan John` (please open another PR; I'll be merging this one).